heapsort.java

```diff
@@ -0,0 +1,74 @@
+// Java program for implementation of Heap Sort 
+public class HeapSort 

```

Comment:
Codacy found an issue: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6319142)